finance.tex

```diff
@@ -1,2 +1,145 @@
+distribution within one week.
+If it is approved, the alternative percentage distribution will be discussed by
+the Executive Board and presented to the chapter with the next semester’s
+budget.
+It will be approved or denied by the chapter as a part of the next semester’s
+budget.
+
+\subsection{Prohibited Uses of Budgeted Funds}
+
+Funds allocated in the manner described in this section shall not be used for
+any Prohibited Use unless either a) such use is clearly denoted in the budget
+approved by the chapter and this Sub-Section was specifically referenced when
+the budget was debated by the chapter, as reflected in the minutes; or b) such
+use is approved by a super-majority vote of the chapter after the budget is
+passed.
+The Prohibited Uses are: 
+\begin{itemize}
+    \item donation of the funds
+\end{itemize}
+
+\section{Reserve Funds}
+
+Each committee, with the exception of Risk Management, shall have a reserve
+fund in order to offset any unexpected costs.
+The Executive Board may decide via the voting procedure to add money to
+individual Reserve Funds from the chapter’s General Savings Fund.
+In addition, any money allocated to a Committee but not spent by the end of the
+semester will be automatically added to that Committee’s Reserve Fund.
+The amount of money in each Reserve Fund can never exceed \$250; any money that
+would be deposited in a Reserve Fund past \$250 will instead be put into the
+chapter’s General Savings Fund.
+
+\section{General Savings}
+
+For money to be taken out of the chapter’s General Savings Fund, the proposed
+expenditure must be presented to, and approved by the Finance Committee.
+If the President or Finance Chairman determine the situation to be an
+emergency, and the Finance Committee will not meet before the expenditure must
+be made, the Finance Chairman and President may decide to approve or deny the
+expenditure independent of the Finance Committee.
+If this occurs, the expenditure must be disclosed to the Finance Committee as
+soon as possible.
+
+\section{Reimbursements}
+
+Reimbursements for individual brothers are at the discretion of the Finance
+Chairman.
+Reimbursements may be distributed by check or Venmo.
+The Finance Chairman shall make his best effort to distribute approved
+reimbursements within one week of their request.
+
+\subsection{Reimbursement Denial Appeals}
+
+If a brother believes he deserves a reimbursement for a legitimate expenditure,
+and the Finance Chairman denies him a reimbursement, he may appeal the denial.
+The brother may appeal to the Kai Cabinet.
+The Kai Cabinet shall hear the brother’s case, including input from the Finance
+Committee, and vote to determine whether the brother deserves a reimbursement.
+If the reimbursement is approved by the Kai Cabinet, the Finance Chairman shall
+give the reimbursement to the brother within one week.
+
+\section{Venmo}
+
+The official Venmo account of the Eta Tau chapter of Beta Theta Pi shall be
+@BetaThetaPiWPI.
```

Comment:
I would make this monospaced like `\texttt{@BetaThetaPiWPI}` since it's a computer identifier and not a real word

Reply:
👍